downloads/swan-lake-release-notes/swan-lake-preview4/RELEASE_NOTE.md

```diff
@@ -314,6 +315,10 @@ Executing a `fail` Statement will cause the control flow to transfer to the `on
     }
 ```
 
+##### Predeclared module prefixes
+
+The module prefix `t` (which is one of the `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc` or `xml`) has been predeclared as referring to the `lang.t` lang library module now.
```

Comment:
```suggestion
The module prefix `t` (which is one of the `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml` types) has been predeclared as referring to the `lang.t` lang library module now.
```

Reply:
These are not types here, they represent the prefixes/names of the langlib modules.

Reply:
Added the new suggestion in https://github.com/ballerina-platform/ballerina-dev-website/pull/1375#discussion_r503027050.

Reply:
Fixed.